### PR TITLE
Dev default gameboard

### PR DESCRIPTION
**Pull request by PalumboN**
Adding some defaults for Wollok-Game:

- Gameboard title = "Wollok Game"
- Gameboard dimensions  = 5 x 5 cells
- Gameboard background
- Visual wko's image

### CODE REVIEW

**Comment by PalumboN** [comment]
Close https://github.com/uqbar-project/wollok/issues/493

File: org.uqbar.project.wollok.lib/src/org/uqbar/project/wollok/lib/WollokConventionExtensions.xtend [hunk 1]
```diff
@@ -15,27 +18,30 @@ class WollokConventionExtensions {
...
 		instanceVariables.entrySet.filter[key.printableVariable]
 	}
 	
-	
 	def static findConvention(WollokObject it, List<String> conventions) {
-		var getter = allMethods.map[it.name].findFirst[isGetter(conventions)]
-		if (getter != null)
-			return call(getter)
-
+		var WollokObject value = null
```
Comment (charlyraffellini) [inline org.uqbar.project.wollok.lib/src/org/uqbar/project/wollok/lib/WollokConventionExtensions.xtend]: What do you think about this?

```xtend
def static findConvention(WollokObject it, List<String> conventions) {
  var getter = allMethods.map[it.name].findFirst[isGetter(conventions)]
  if (getter != null)
    return Optional.ofNullable(call(getter))
  var WollokObject value = conventions.map[c|instanceVariables.get(c)].filterNull.head
  Optional.ofNullable(value)
}
```

**Comment by Juancete** [comment]
:cocktail: :+1:

File: org.uqbar.project.wollok.lib/src/org/uqbar/project/wollok/lib/WollokConventionExtensions.xtend (hunk 1 above)
Comment (PalumboN) [inline org.uqbar.project.wollok.lib/src/org/uqbar/project/wollok/lib/WollokConventionExtensions.xtend]: Like!

**Pull request merged**